src/main/kotlin/no/nav/hjelpemidler/oppgave/domain/BehovsmeldingData.kt

```diff
@@ -0,0 +1,18 @@
+package no.nav.hjelpemidler.oppgave.domain
+
+import com.fasterxml.jackson.databind.JsonNode
+import java.util.UUID
+
+data class BehovsmeldingData(
```

Comment:
Dette er vel et use case for å bruke hm-behovsmeldingsmodell eller ville dere slippe å ta inn hele den for dette ene feltet kanskje?